homeassistant/components/whirlpool/sensor.py

```diff
@@ -119,11 +119,12 @@ class WhirlpoolSensorEntityDescription(
         key="DispenseLevel",
         name="Detergent Level",
         translation_key="whirlpool_tank",
+        entity_registry_enabled_default=False,
         device_class=SensorDeviceClass.ENUM,
         options=list(TANK_FILL.values()),
-        value_fn=lambda WasherDryer: TANK_FILL[
-            WasherDryer.get_attribute("WashCavity_OpStatusBulkDispense1Level")
-        ],
+        value_fn=lambda WasherDryer: TANK_FILL.get(
+            WasherDryer.get_attribute("WashCavity_OpStatusBulkDispense1Level"), None
```

Comment:
by the way, in the future I think we should really avoid using get_attribute here, and have proper getters on the library's classes. HA should not have to know about WashCavity_OpStatusBulkDispense1Level